pkgs/development/python-modules/jax/default.nix

```diff
@@ -38,7 +38,7 @@ buildPythonPackage rec {
     owner = "google";
     repo = "jax";
     # google/jax contains tags for jax and jaxlib. Only use jax tags!
-    rev = "refs/tags/jaxlib-v${version}";
+    rev = "refs/tags/jax-v${version}";
     hash = "sha256-poQQo2ZgEhPYzK3aCs+BjaHTNZbezJAECd+HOdY1Yok=";
```

Comment:
Note to self: updating the hash here is not necessary since both `jax-v0.4.25` and `jaxlib-v0.4.24` tags point to https://github.com/google/jax/commit/f9e20d58754283de87b2ed35cc9df58bcdff2073 at the time of writing.